crates/e2e/sandbox/Cargo.toml

```
@@ -0,0 +1,41 @@
+[package]
+name = "ink_sandbox"
+version.workspace = true
+authors.workspace = true
```

Part of this code is extracted, right? Then we should put:

```suggestion
authors = ["Cardinal Cryptography", "Parity Technologies <admin@parity.io>"]
```

The code licenses for ink! and DRink! are both Apache 2.0, so no issue there.